cli.go

```diff
@@ -148,6 +150,12 @@ func (cli *CLI) Run(args []string) int {
 
 		p := printer.NewPrinter(cli.outStream, cli.errStream)
 		p.Print(issues, format)
+
+		if exitErrorStatus {
+			if len(issues) > 0 {
+				return ExitCodeError
```

Comment:
Please use another status code. (ex: `ExitCodeIssuesFound`, etc.) `ExitCodeError` is reserved for execution error.